packages/manager/modules/pci/src/projects/project/storages/containers/add/add.controller.js

```diff
@@ -0,0 +1,131 @@
+import get from 'lodash/get';
+import map from 'lodash/map';
+
+import Container from '../container.class';
+
+import {
+  OBJECT_CONTAINER_TYPES,
+} from '../containers.constants';
+
+export default class PciStoragesContainersAddController {
+  /* @ngInject */
+  constructor(
+    $q,
+    $timeout,
+    $translate,
+    CucCloudMessage,
+    PciProjectStorageBlockService,
+    PciProjectStorageContainersService,
+  ) {
+    this.$q = $q;
+    this.$timeout = $timeout;
+    this.$translate = $translate;
+    this.CucCloudMessage = CucCloudMessage;
+    this.PciProjectStorageBlockService = PciProjectStorageBlockService;
+    this.PciProjectStorageContainersService = PciProjectStorageContainersService;
+  }
+
+  $onInit() {
+    this.displaySelectedRegion = false;
+    this.displaySelectedType = false;
+
+    this.types = OBJECT_CONTAINER_TYPES;
+    this.typesList = map(
+      this.types,
+      type => ({
+        id: type,
+        name: this.$translate.instant(`pci_projects_project_storages_containers_add_type_${type}_description`),
+      }),
+    );
+
+    this.loadings = {
+      regions: false,
+      types: false,
+      save: false,
+    };
+
+    this.container = new Container({
+      archive: this.archive,
+    });
+
+    this.loadings.regions = true;
+
+    this.$translate.refresh()
+      .then(() => this.loadMessages())
+      .then(() => this.PciProjectStorageBlockService.getAvailablesRegions(this.projectId))
+      .then((regions) => {
+        this.regions = regions;
+      })
+      .catch(err => this.CucCloudMessage.error(
```

Comment:
```suggestion
      .catch(err => {
        this.CucCloudMessage.error(
```
It doesn't matter much but `error` returns `undefined` so this could be potentially confusing to think that a useful value is returned